dashboard/app/models/script.rb

```diff
@@ -24,6 +24,7 @@
 
 require 'cdo/script_constants'
 require 'cdo/shared_constants'
+require 'script_lessons_serializer'
```

Comment:
we have not yet committed to renaming "script" to "unit", though I'd really like to. What is your opinion on how much work this (the use of "script" throughout this PR) would add to such a rename?

I do not think I see any use of the word "script" here that could not be fixed in a single code change, e.g. you are not introducing any dependency on a DB table or column or other thing which would require a multi-step process in order to rename later. So I don't think makes a rename too much harder, but please correct me if you see something I am missing.

Reply:
I agree with your assessment - I don't think this adds much work to that. Even if the word `script` is in the file extension - which it doesn't have to be - those could be changed in bulk in one change.